ch02.adoc

```diff
@@ -54,7 +54,8 @@ The examples in this document that use string-valued variables alternate between
 === Naming Conventions
 
 Variable, dimension, attribute and group names should begin with a letter and be composed of letters, digits, and underscores.
-Note that this is in conformance with the COARDS conventions, but is more restrictive than the netCDF interface which allows use of the hyphen character.
+By the word _letters_ we mean the standard ASCII letters uppercase `A` to `Z` and lowercase `a` to `z`.
```

Comment:
Digits probably needs to be clarified as well since Unicode has hundreds of digits and numbers outside of 0-9. Maybe just adding sentence specifying digits below this one:

```suggestion
By the word _letters_ we mean the standard ASCII letters uppercase `A` to `Z` and lowercase `a` to `z`.
By the word _digits_ we mean the standard ASCII digits `0` to `9`.
```